hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -130,11 +131,13 @@ private OzoneManagerRatisServer(Configuration conf,
     LOG.info("Instantiating OM Ratis server with GroupID: {} and " +
         "Raft Peers: {}", raftGroupIdStr, raftPeersStr.toString().substring(2));
 
+    this.omStateMachine = getStateMachine(this.raftGroupId);
```

Comment:
Here, we are passing raftGroupId to getStateMachine, but we are not using that param in method.